includes/utils.php

```diff
@@ -1182,3 +1182,14 @@ function remote_http_request( $url, $args = array(), $fallback = '', $threshold
 
 	return wp_remote_request( $url, $args );
 }
+
+/**
+ * Determines if a post is distributed.
+ *
+ * @param int $post_id The post ID.
+ * @return bool True if the post is distributed, false otherwise.
+ */
+function get_is_distributed_post( $post_id ) {
```

Comment:
```suggestion
 * @param int|\WP_Post $post The post object or ID been checked.
 * @return bool True if the post is distributed, false otherwise.
 */
function is_distributed_post( $post ) {
	$post = get_post( $post );
	if ( ! $post ) {
		return false;
	}
	$post_id = $post->ID;
```

This does a couple of things:
* removes `get_` prefix, `is_` is a fairly common prefix in the plugin already
* accepts either a post object or ID, this is a common practice in core so I like to copy it

Reply:
Thanks @peterwilsoncc . Updated.